### PR TITLE
Revisiting Rock Paper Scissors lesson: Fix typo

### PR DESCRIPTION
**Pull request by sujkovic**
Remove the word 'to' from the last bullet point of additional resources

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

<--
This fix removes the word "to" in the last bullet point of the additional resources section of the lesson because it sounds like a typo.
-->

**2. This PR:**

<--
-Remove the word "to" from last bullet of additional resources.
-->

**3. Additional Information:**

### CODE REVIEW

File: foundations/javascript_basics/revisiting_rock_paper_scissors.md [hunk 1]
```diff
@@ -66,4 +66,4 @@ This section contains helpful links to related content. It isn’t required, so
...
 *   Make pushing your local commits to remote branches **easier** with the command `git push -u origin <branch>`. It automatically links the local branch you push with the remote one. [Read this educative.io article](https://www.educative.io/edpresso/what-is-the-git-push--u-remote-branch-name-command) by Talha Ashar and commit faster to a remote branch with a simple `git push` command.
-*   Learn Git Branching by going through this [interactive visualizer](https://learngitbranching.js.org/) by Peter Cottle. You can learn the substantial commands about branching while seeing how the commands you write affects to your visually presented branch tree.
+*   Learn Git Branching by going through this [interactive visualizer](https://learngitbranching.js.org/) by Peter Cottle. You can learn the substantial commands about branching while seeing how the commands you write affects your visually presented branch tree.
```
Comment (xandora) [inline foundations/javascript_basics/revisiting_rock_paper_scissors.md:69]:
```suggestion
*   Learn Git Branching by going through this [interactive visualizer](https://learngitbranching.js.org/) by Peter Cottle. You can learn the substantial commands about branching while seeing how the commands you write affect your visually presented branch tree.
```

**Review by xandora — CHANGES_REQUESTED** [review]
Looks good, one other small adjustment, then we can merge!

**Review by sujkovic — COMMENTED** [review]
Sounds good to me!

**Review by xandora — APPROVED** [review]
Looks good to me :)

**Pull request merged**